conda/recipes/versions.yaml

```diff
@@ -86,8 +86,14 @@ holoviews_version:
   - '>=1.14.8,<=1.15.3'
 ipython_version:
   - '=7.31.1'
+# TODO Update ipykernel_version once https://github.com/jupyter-server/jupyter_server/issues/1198 is resolved
+ipykernel_version:
+  - '=6.20.2'
 isort_version:
   - '=5.12.0'
+# TODO Update jupyter_core_version once https://github.com/jupyter-server/jupyter_server/issues/1198 is resolved
+jupyter_core_version:
+  - '=5.1.5'
```

Comment:
From my testing this seemed unnecessary, just `ipykernel` was needed. I doesn't seem to harm either though, so I'll leave it up to you.

Reply:
I think I want to be overly cautious here since we are so close to the release.